pyproject.toml

```diff
@@ -1,3 +1,86 @@
+[tool.poetry]
+name = "fedora_messaging"
+version = "3.4.1"
+description = "A set of tools for using Fedora's messaging infrastructure"
+authors = [
+    "Fedora Infrastructure <admin@fedoraproject.org>"
+]
+license = "GPLv2+"
+readme = "README.rst"
+repository = "https://github.com/fedora-infra/fedora-messaging"
+maintainers = ["Fedora Infrastructure Team"]
+classifiers = [
+        "Development Status :: 5 - Production/Stable",
+        "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+    ]
+keywords = ["fedora"]
+packages = [
+    { include = "fedora_messaging" },
+]
+include = [
+    { path = "docs", format = "sdist" },
+		{ path = "CODEWNERS", format = "sdist"},
+		{ path = "configs", format = "sdist"},
+		{ path = "*.service", format = "sdist"},
+]
+
+[tool.poetry.dependencies]
+python = ">=3.9"
+blinker = "*"
+click = "^8.1.7"
+crochet = "^2.1.1"
+jsonschema = "^4.19.0"
+twisted = "^22.10.0"
+pytz = "^2023.3"
+pyopenssl = "^23.2.0"
+pika = ">=1.0.1"
```

Comment:
Here we could have `^1.0.1` instead, it would save us from breaking when pika 2.0.0 comes out